src/valhalla_pack_elevation.cc

```diff
@@ -1,187 +0,0 @@
-#include <cstdint>
```

Comment:
we remove support for lz4. while it is a cool thing to have it didnt end up being all that much more useful than gzip. while it is no doubt faster than gzip its still slow enough that using it in a realtime application is pretty much in bearable (for server applications)